torchao/quantization/qat/README.md

```diff
@@ -115,11 +115,20 @@ To fake quantize embedding in addition to linear, you can additionally call
 the following with a filter function during the prepare step:
 
```

Comment:
There's a section below that says "torchtune integration". Can you add a similar section for "Axolotl Integration" and add a small code snippet there?